spark/Debugging-extended-Spark-History-Server.md

```diff
@@ -0,0 +1,202 @@
+Extended Spark History Server is built on top of the community version of Spark History Server and provide advanced features. The current extension includes data tab and graph tab. Data tab shows the input/output data source of the Spark job including table operations. This allows users to validate the output, preview input schema/data to facilitate the development. Graph tab provides job graph, playback and heatmap. This helps users to understand the bottleneck, monitor and diagonose issues. Please check [Extended Spark UI documentation](https://go.microsoft.com/fwlink/?linkid=2005401) for detail information.
+
+## 1. Hotfix extended Spark History Server
```

Comment:
Provide a link to an script stored in https://github.com/hdinsight/script-actions for install and uninstall

Reply:
Thanks for the feedback! Create PR below to upload to script-actions. Will fix link once that PR is merged:
https://github.com/hdinsight/script-actions/pull/13